src/constants/common.ts

```diff
@@ -1,14 +1,17 @@
-import type { GithubDeploymentStatusState } from '../types';
+import type { GithubDeploymentStatusState } from '@/types';
 
 /**
  * 액션에서 필수적으로 사용되는 입력되어야 할 키 값입니다.
  * **/
-const ACTION_REQUIRED_INPUT_KEY = [
+const ACTION_REQUIRED_INPUT_KEY_LIST = [
   'token', // GitHub Token
-  'extractionPoint', // 본문을 추출할 기점 해당 기점 문자열 ex) "## 리뷰 요약 정보"
+  'extractionStartPoint', // 본문을 추출할 기점 해당 기점 문자열 ex) "## 리뷰 요약 정보"
   'slackWebhookURL', // Slack Webhook URL
+  'projectConfig', //
 ] as const;
 
+const ACTION_INPUT_KEY_LIST = [...ACTION_REQUIRED_INPUT_KEY_LIST, 'extractionEndPoint'] as const;
```

Comment:
💊 
이렇게 분리된다면.. optional 상수도 분리해두는게 깔끔하겠네요.

```suggestion
const ACTION_REQUIRED_INPUT_KEY_LIST = [
  'token', // GitHub Token
  'extractionStartPoint', // 본문을 추출할 시작 지점에 해당되는 문자열 ex) "## 리뷰 요약 정보"
  'slackWebhookURL', // Slack Webhook URL
  'projectConfig', //
] as const;

const ACTION_OPTIONAL_INPUT_KEY_LIST = [
  'extractionEndPoint', 본문을 추출할 종료 지점에 해당되는 문자열 ex) "## 리뷰 요약 정보"
] as const;

const ACTION_INPUT_KEY_LIST = [...ACTION_REQUIRED_INPUT_KEY_LIST, ...ACTION_OPTIONAL_INPUT_KEY_LIST] as const;
```

Reply:
이 부분은 이미 대응이 되어 있었는데 커밋 히스토리에는 안보이네요...?

![image](https://github.com/user-attachments/assets/e2ee1c24-ca7a-45bc-aa9d-d1f843ab78b1)